controllers/imagepolicy_controller.go

```diff
@@ -89,19 +104,23 @@ func (r *ImagePolicyReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 
 	policy := pol.Spec.Policy
 
-	switch {
-	case policy.SemVer != nil:
-		latest, err := r.calculateLatestImageSemver(&policy, repo.Status.CanonicalImageName)
-		if err != nil {
+	if policy.SemVer == nil {
```

Comment:
The reason this is a `switch` is that there are likely to be other kinds of policy, imminently. When that's the case, it'll just be rewritten back again -- I'd prefer to just keep the current form and avoid that.

Reply:
I understand, thanks!